app/helpers/orchid/facet_helper.rb

```diff
@@ -12,10 +12,10 @@ def create_label(key, default="No Label")
   end
 
   # type: type of facet (category, format, etc)
-  # value: the normalized value ("willa cather", "Yellowstone Kelly")
-  #        used for URL creation and translation matching
+  # normalized: the normalized value ("willa cather", "Yellowstone Kelly")
+  #     used for URL creation and translation matching
   # label: non normalized value ("Willa Cather", '"Yellowstone Kelly"')
-  def facet_label(type: type, normalized: normalized, label: nil)
+  def facet_label(type: nil, normalized: nil, label: nil)
```

Comment:
Ah, I didn't realize that was a specific thing in RuboCop. I feel like I may have committed that foul in the past. But good to know going forward :)